src/TimelineItem.cc

```diff
@@ -396,8 +396,8 @@ TimelineItem::setupAvatarLayout(const QString &userName)
 
         userAvatar_ = new Avatar(this);
         userAvatar_->setLetter(QChar(userName[0]).toUpper());
-        userAvatar_->setBackgroundColor(QColor("#eee"));
-        userAvatar_->setTextColor(QColor("black"));
+        //        userAvatar_->setBackgroundColor(QColor("#eee"));
```

Comment:
We also need a color for the background, otherwise it's the same as the timeline and only a letter is visible.